### DISCUSSION TITLE
Setpoint is missing min/max values

### DISCUSSION THREAD

**Discussion by alex3305** [zwave-js/zwave-js#7202] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

**What is happening?**
Since [Home Assistant 2024.8](https://www.home-assistant.io/blog/2024/08/07/release-20248/) Home Assistant validates temperatures set within climate devices. This however has broken some devices. One of them is my Popp TRV (also known as Danfoss LC-13 or Devolo TRV). This devices has known setpoint limits from 4-28°C, but Home Assistant uses the defaults 7-35°C because Z-WaveJS doesn't report this metadata.

In the past this has worked through Customization within Home Assistant.

**What did you expect to happen instead?**
That Z-WaveJS reports the (known) metadata on devices that doesn't report these values themselves.

**Steps to reproduce the behavior:**
  1. Set the temperature to anything below 7°C from Home Assistant
  2. Get an error

**Anything else we should know?**
I know this probably isn't a Z-WaveJS issue, but it also isn't a Home Assistant issue either. So I'm testing the waters to know where this can be fixed. Because my house is dependent on these TRV's.

- [Home Assistant - Core issue #123612](https://github.com/home-assistant/core/issues/123612)
- [Home Assistant - PR #126924 (closed)](https://github.com/home-assistant/core/pull/126924)

### Software versions

Driver (node-zwave-js): 13.2.0

Z-Wave JS UI: 1.38.0

Home Assistant Z-Wave Integration: 2024.9

### Z-Wave Controller (Stick/Dongle/...)

Z-Wave.me UZB1

### Device information

Manufacturer: Popp
Model name: 010101

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

n/a

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment]
We'll discuss this internally. `node-zwave-js` does not add synthetic capabilities to devices, so it can't really be fixed here. HA's behavior of validating setpoints against an _assumed_ range seems wrong to me, and actually violates Z-Wave certification requirements:
![grafik](https://github.com/user-attachments/assets/9afdf969-daed-445b-8b26-62e942cf173c)
